### PR TITLE
Partridge calculations

### PR DESCRIPTION
**Pull request by lmcewanhmrc**
Nav and CYA changes for single year claim

### CODE REVIEW

File: test/models/returns/ClaimedCreditsDetailsSpec.scala [hunk 1]
```diff
@@ -16,48 +16,44 @@
...
 
           val credits = CreditsClaimedDetails(newAns, CreditBalance(10, 4, 200, true))
 
           credits.summaryList mustBe Seq(
             CreditExportedAnswerPartialKey -> (if (exported) "site.yes" else "site.no"),
-            CreditExportedWeightPartialKey -> exportedWeight.asKg,
+            exportedWeight.fold("N/A" -> "N/A")(o => CreditExportedWeightPartialKey -> o.asKg),
             CreditConvertedAnswerPartialKey -> (if (converted) "site.yes" else "site.no"),
-            CreditConvertedWeightPartialKey -> convertedWeight.asKg,
+            convertedWeight.fold("N/A" -> "N/A")(o => CreditConvertedWeightPartialKey -> o.asKg),
```
Comment (liampan) [inline test/models/returns/ClaimedCreditsDetailsSpec.scala:56]: is it "N/A" here or the weight? is it worth just doing a .get to make the test more deterministic?

File: app/pages/returns/credits/ConvertedCreditsPage.scala [hunk 2]
```diff
@@ -16,9 +16,14 @@
 
 package pages.returns.credits
 
+import models.returns.CreditsAnswer
 import pages.QuestionPage
 import play.api.libs.json.JsPath
 
+case object OldConvertedCreditsPage extends QuestionPage[CreditsAnswer] {
+  override def path: JsPath = JsPath \ "convertedCredits"
```
Comment (liampan) [inline app/pages/returns/credits/ConvertedCreditsPage.scala:24]: these now clash is that ok?

**Review by liampan — APPROVED** [review]

File: app/pages/returns/credits/ConvertedCreditsPage.scala (hunk 2 above)
Comment (liampan) [reply app/pages/returns/credits/ConvertedCreditsPage.scala:24]: ignore

File: test/models/returns/ClaimedCreditsDetailsSpec.scala (hunk 1 above)
Comment (lmcewanhmrc) [reply test/models/returns/ClaimedCreditsDetailsSpec.scala:56]: to-do list 👍

**Pull request merged**